language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionUtil.java

```diff
@@ -865,6 +889,27 @@ private static String generateIfElseText(String varName, String spaces, String p
         return LINE_SEPARATOR + newTextBuilder.toString();
     }
 
+    private static String generateGetterFunctionBodyText(String varName, String typeName, String spaces) {
+        StringBuilder newTextBuilder = new StringBuilder();
+        String functionName = varName.substring(0, 1).toUpperCase(Locale.ROOT) + varName.substring(1);
+        newTextBuilder.append(String.format
+                (LINE_SEPARATOR + LINE_SEPARATOR + spaces + "public function get%s() returns %s{ " + LINE_SEPARATOR +
+                spaces + spaces + "return self.%s;" + LINE_SEPARATOR + spaces + "}" ,
+                functionName, typeName, varName));
+        return newTextBuilder.toString();
+    }
+
+    private static String generateSetterFunctionBodyText(String varName, String typeName, String spaces) {
+
+        StringBuilder newTextBuilder = new StringBuilder();
+        String functionName = varName.substring(0, 1).toUpperCase(Locale.ROOT) + varName.substring(1);
+        newTextBuilder.append(String.format
+                (LINE_SEPARATOR + LINE_SEPARATOR + spaces + "public function set%s(%s %s) { " + LINE_SEPARATOR +
+                spaces + spaces + "self.%s = %s;" + LINE_SEPARATOR + "" + spaces + "}" ,
+                functionName, typeName, varName, varName, varName));
```

Comment:
Shall we append the strings separately?

Reply:
Updated in 9b8ee2e211c6ab47082d00b3379ae13676eb610d